UnanimusProject/src/com/sjgilbert/unanimus/GroupActivity.java

```diff
@@ -78,8 +111,22 @@ protected void onCreate(Bundle savedInstanceState) {
             @Override
             public void onClick(View v) {
                 Intent intent = new Intent(GroupActivity.this, VotingActivity.class);
+                intent.putExtra("groupID", groupName);
```

Comment:
The string "groupID" should be a package-local static final field of GroupActivity.